cloudbuild.yaml

```diff
@@ -0,0 +1,32 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+timeout: 1200s
+steps:
+  - name: gcr.io/cloud-builders/docker
```

Comment:
Is this conventional? I noticed some repos used `gcr.io/k8s-staging-test-infra/gcb-docker-gcloud`.

Reply:
> If your build process is driven by a Makefile or similar, you can use GCB to invoke that. We provide the gcr.io/k8s-testimages/gcb-docker-gcloud image, which contains components that are likely to be useful for your builds. 
https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md#makefile-build-example

So, `gcr.io/k8s-staging-test-infra/gcb-docker-gcloud` image is for Makefile build, maybe...? 

Because simple build example uses `gcr.io/cloud-builders/docker` (and I followed it.)
https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md#simple-build-example